cfavml/src/danger/impl_avx2fma.rs

```diff
@@ -225,3 +281,58 @@ impl SimdRegister<f64> for Avx2Fma {
         Avx2::write(mem, reg)
     }
 }
+
+impl Hypot<f64> for Avx2Fma {
+    #[inline(always)]
+    unsafe fn hypot(x: Self::Register, y: Self::Register) -> Self::Register {
+        // convert the inputs to absolute values
+        let (x, y) = (
+            _mm256_andnot_pd(_mm256_set1_pd(-0.0), x),
+            _mm256_andnot_pd(_mm256_set1_pd(-0.0), y),
+        );
+        // find the max and min of the two inputs
+        let (hi, lo) = (
+            <Avx2 as SimdRegister<f64>>::max(x, y),
+            <Avx2 as SimdRegister<f64>>::min(x, y),
+        );
```

Comment:
Same vibe as mentioned earlier, best to have these just as two separate lines rather than a tuple.